psrm_utils.py

```diff
@@ -15,6 +18,24 @@ def cache_psrm(cache='psrm.pkl', force=False, *args, **kwargs):
             psrm = pickle.load(f)
     return psrm
 
+
+def load_dw_rpt(path):
+    """Load weird rpt from DW export."""
+    with open(path, 'r') as f:
+        lines = f.readlines()
+    # custom header for now
+    header = ['FordringID', 'FordringSaldo', 'InddrivelsesrenterAkk']
```

Comment:
Hvorfor ikke bare kalde `FordringID` for `NYMFID` her så du er fri for at rename senere?

Reply:
Det gik lige lidt hurtigt, da vi skulle hurtigt igang. Men ideen er at loaded headeren automatisk og så rename.